cli/src/main/java/io/aklivity/zillabase/cli/config/ZillabaseStudioConfig.java

```diff
@@ -21,10 +21,10 @@ public final class ZillabaseStudioConfig
     public static final String DEFAULT_STUDIO_TAG = Zillabase.version();
     public static final int DEFAULT_STUDIO_HTTP_PORT = 7194;
 
-    public String tag = DEFAULT_STUDIO_TAG;
-    public int port = DEFAULT_STUDIO_HTTP_PORT;
+    private final String tag = DEFAULT_STUDIO_TAG;
+    private final int port = DEFAULT_STUDIO_HTTP_PORT;
```

Comment:
Please revert this encapsulation methods, let's keep the code consistent with the other configs.
Instead, if you need it to be readonly, then `public final ...` is okay.